src/UbiHttp.h

```diff
@@ -24,16 +24,15 @@ Developed and maintained by Jose Garcia for IoT Services Inc
 #ifndef _UbiHttp_H_
 #define _UbiHttp_H_
 
+#include "UbiConstants.h"
```

Comment:
It seems that this include is not used.

Reply:
it is used by UbiHttp.cpp, which import this header